r/R/dplyr-functions.R

```diff
@@ -847,6 +847,14 @@ agg_funcs$var <- function(x, na.rm = FALSE, ddof = 1) {
     options = list(skip_nulls = na.rm, min_count = 0L, ddof = ddof)
   )
 }
+agg_funcs$median <- function(x, na.rm = FALSE) {
```

Comment:
Can you also add `quantile`, and just error if there is more than one quantile specified?

Reply:
Done